tests/test_equations.py

```diff
@@ -51,8 +51,9 @@ class t(Variable):
 def test_equation():
     """Test variable definition."""
     assert demo_fall.__doc__ == demo_fall.definition.__doc__
-    assert demo_fall.subs(Variable.__defaults__
-                          ).evalf(subs={demo_fall.definition.t: 1}) == 4.9
+
+    assert demo_fall.rhs.subs(Variable.__defaults__
```

Comment:
`demo_fall.rhs.subs` -> `demo_fall.subs` should work anyway

Reply:
You are right, subs is now tested in `test_substitution()`, while evalf already behaves like in Sympy for equalities:
```python
from sympy import Symbol, Eq
x = Symbol('x')
y = Symbol('y')
test = Eq(y, x**2)
print test.evalf(subs={x: 2})
print test.rhs.evalf(subs={x: 2})
```
returns
```
Eq(y, x**2)
4.00000000000000
```
Strangely, the substituion within evalf is not applied in an equality, so I will remove this test.